test/merge-coverage.js

```diff
@@ -0,0 +1,245 @@
+const fs = require('fs');
```

Comment:
If I'm understanding this correctly, does this combine coverage reports from multiple distinct runs of `jest` into one report? If so, this is super rad and I think we could find this immensely useful on the monorepo side.

Reply:
Yep it sure does.

Reply:
and mocha :D cause they both use nyc